src/main/java/com/openlattice/edm/PostgresEdmManager.java

```diff
@@ -143,68 +161,102 @@ public void revoke(
 
     }
 
-    private String grantOnTable( String table, String principalId, String permission ) {
-        return String.format( "GRANT %s ON TABLE %s TO %s", permission, table, principalId );
-    }
-
-    private String renameColumn( String table, String current, String update ) {
-        return String.format( "ALTER TABLE %s RENAME COLUMN %s TO %s", table, current, update );
-    }
+    public Iterable<EntitySet> getAllEntitySets() {
```

Comment:
This should be switched to using PostgresIterable, but is outside the scope of this PR.